### ISSUE TITLE
Train details for RF100-VL

### ISSUE THREAD

**Issue by capsule2077** [roboflow/rf-detr#266]
Dear authors,

Thank you for your great work and for sharing the codebase.

I'm currently trying to reproduce the results for the **RF100-VL** experiments as reported in your paper, but I couldn’t find the detailed hyperparameter settings (e.g., learning rate, batch size, training epochs) specific to this variant in the repository or the supplementary materials.

Could you kindly provide the configuration or training script used for RF100-VL(such as yolov8 yolov11)?

**Comment by isaacrob-roboflow** [comment]
This contains the code used to generate the YOLO numbers https://github.com/roboflow/semi_supervised_benchmark

The RF-DETR numbers for the current release are using the default arguments but with batch size 16 and grad accum 1 .. we'll be checking for any bugs in the next weeks, but the numbers did come from a private fork of this repo

**Comment by capsule2077** [comment]
@isaacrob-roboflow 
Hi,
I tried to reproduce the RF100VL results using the RFDETR codebase with the default settings. I trained each dataset for 12 epochs and did not modify any other parameters(batch size 16 and grad accum 1). However, the results I obtained are significantly lower than those reported.

Is there something I might have missed in the setup or configuration? Any help would be appreciated.

Thanks!

**Comment by isaacrob-roboflow** [comment]
yep! couple things. all models are trained for 100 epochs, not 12. there's also a bug we're addressing where the head is not initialized correctly (cc @probicheaux who is working on the fix) .. finally, we trained from our o365 checkpoints which we haven't released publicly yet 

a preprint will come out soon with more details and we're working to tighten up the open source implementation
